metricbeat/module/elasticsearch/node_stats/_meta/fields.yml

```diff
@@ -312,4 +312,21 @@
               type: long
             - name: rejected.count
               type: long
-
+    - name: indexing_pressure
```

Comment:
This entire group must not contain `elasticsearch.node.stats` as a prefix of each field and it must follow naming conventions

Reply:
This appears to have been resolved by more recent commits.